waiter/src/waiter/handler.clj

```diff
@@ -1343,12 +1343,13 @@
                     :existing-owner run-as-user
                     :log-level :info
                     :service-id service-id
+                    :instance-id instance-id
                     :status http-403-forbidden}))
         (async/go
           (let [response-chan (async/promise-chan)
             _ (execute-signal
               notify-instance-killed-fn peers-acknowledged-eject-requests-fn scheduler
-              populate-maintainer-chan! timeout-config instance-id service-id (keyword signal-type) timeout
+              populate-maintainer-chan! timeout-config instance-id service-id (keyword signal-type) (if (zero? (Integer/parseInt timeout)) nil (Integer/parseInt timeout))
```

Comment:
We should extract this to timeout-ms and then validate that it is a positive number.
```
(if (str/blank? timeout) nil (Integer/parseInt timeout))
```

Reply:
This is done in the CLI. Are you saying we should also check for positivity here as well?